e2e-tests/IIoTPlatform-E2E-Tests/Discovery/DiscoveryTestTheory.cs

```diff
@@ -32,14 +33,14 @@ public class DiscoveryTestTheory {
         }
 
         [Fact, PriorityOrder(0)]
-        public void Test_Discover_All_OPC_UA_Endpoints() {
+        public async Task Test_Discover_All_OPC_UA_Endpoints() {
             // Add 5 servers
             var endpointUrls = TestHelper.GetSimulatedOpcServerUrls(_context).Take(5).ToList();
-            AddTestOpcServers(endpointUrls);
+            AddTestOpcServers(endpointUrls, _cancellationTokenSource.Token);
 
             // Discover all servers
-            var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
-            dynamic result = TestHelper.Discovery.WaitForDiscoveryToBeCompletedAsync(_context, cts.Token, requestedEndpointUrls: endpointUrls).GetAwaiter().GetResult();
+            dynamic result = await TestHelper.Discovery.WaitForDiscoveryToBeCompletedAsync(
```

Comment:
Could we use `var` instead of `dynamic` here? I think this would be more consistent with the rest of the code.

Reply:
Sure.